src/test/java/uk/gov/companieshouse/filinghistory/consumer/kafka/ConsumerPositiveComprehensiveIT.java

```diff
@@ -68,7 +68,11 @@ public void setup() {
     @ParameterizedTest
     @CsvSource({
             "annotations/annotation",
-            "associated_filing/associated_filing",
+
+            "annual_return/annual_return",
```

Comment:
The naming convention for the other top level transactions are category/form_type. Could this be changed please? I think having child transaction ones in specific folders is fine though.